### PR TITLE
Add digitisation event details

### PR DESCRIPTION
**Pull request by timhodson**
Re-organises the event class page to make it easier to find events for particular product areas.
Adds digitisation events details.
Also includes minor tidying.

### CODE REVIEW

File: topics/event-classes.md [hunk 1]
```diff
@@ -6,32 +6,67 @@ body_class: event-classes
...
+
+&#10033; These events are only available when using [New List View](https://support.talis.com/hc/en-us/articles/360000008189).
+
+## Copyright Clearance
+
+| Class | Description | Timeseries data? | Raw data? |
+| :- | :- | :- | :- |
+|`digitisation.view`| A digitisation was viewed | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-view)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+|`digitisation.print.showModal`| A digitisation print modal was opened | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-print-showmodal)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+|`digitisation.print`| A digitisation was downloaded for printing | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-print)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```
Comment (camillef) [inline topics/event-classes.md]: This is only recorded in Elevate - it is when the user clicks "Print" there, for a Copyright Cleared resource. This is in addition to `player.print`

Comment (camillef) [reply topics/event-classes.md]: In TADC, when the user downloads a digitisation from the print modal, we just record that as a `digitisation.download`. 

It is confusing, I know

File: topics/event-classes.md [hunk 2]
```diff
@@ -6,32 +6,67 @@ body_class: event-classes
...
+|`digitisation.download`| A digitisation was downloaded | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-download)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```
Comment (camillef) [inline topics/event-classes.md]: `digitisation.view` and `digitisation.download` are also recorded by the Elevate player when the content has originated from TADC, in addition to `player.view`, `player.download`

File: topics/event-classes.md [hunk 3]
```diff
@@ -6,32 +6,67 @@ body_class: event-classes
...
+| `player.annotate.createannotation` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createreply` | A new replay (to an existing thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createreply)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.download` | The player resource was downloaded by a user | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-download)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```
Comment (camillef) [inline topics/event-classes.md]: I believe we're missing `player.print`

Comment (camillef) [reply topics/event-classes.md]: (Or does this not make it into Census?)

Comment (camillef) [reply topics/event-classes.md]: Also, are we choosing to ignore the `.contributors` events?

File: topics/event-classes.md (hunk 1 above)
Comment (timhodson) [reply topics/event-classes.md]: Hmm.. that is confusing.

Comment (timhodson) [reply topics/event-classes.md]: Actually you mention player.print. but we don't have an entry for player.print at all.

Comment (timhodson) [reply topics/event-classes.md]: in fact player.print doesn't appear in the census event config at all. Should it?

Comment (camillef) [reply topics/event-classes.md]: I'm not really sure - who made the decision on which Elevate events we send through?

File: topics/event-classes.md (hunk 3 above)
Comment (timhodson) [reply topics/event-classes.md]: What are the `.contributors` events? I see some for some of the events...
What do they do and how are they used?
Actually maybe that is a question for another time.

File: topics/event-classes.md (hunk 2 above)
Comment (timhodson) [reply topics/event-classes.md]: @camillef I think I have clarified this now, but would appreciate another look from yourself.

File: stylesheets/styles.css [hunk 4]
```diff
@@ -363,8 +363,8 @@ abbr {
   }
 
   nav {
-    -webkit-flex: 0 0 35%;
-            flex: 0 0 35%;
+    -webkit-flex: 0 0 25%;
```
Comment (timhodson) [inline stylesheets/styles.css]: Give our content some more room.

File: topics/event-classes.md [hunk 5]
```diff
@@ -6,32 +6,69 @@ body_class: event-classes
...
-| `addToList` | An item was added to a list |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#addToList)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `addToList` <span class="deprecated">DEPRECATED</span> | An item was added to a list |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#addToList)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
 | `list.publish` | A list was published |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#list-publish)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
-| `player.timer.2` | A resource was viewed by a student in the player for an amount of time |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-timer-2)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```
Comment (timhodson) [inline topics/event-classes.md]: These have been reorganised...

File: topics/hierarchy-descendants.md [hunk 6]
```diff
@@ -39,4 +39,4 @@ and that this view only supports up to five levels of depth:
 | `url` | `VARCHAR(4096)` | Canonical URL of the hierarchy node |  |
 | `descendant_url` | `VARCHAR(4096)` | Canonical URL of the descendant node  |  |
 
-**WARNING:** Because this is a beta service, the data dictionary is subject to change. For that reason always refer to column names directly in your SQL statements, not column positions.
+**WARNING:** The data dictionary is subject to change. For that reason always refer to column names directly in your SQL statements, not column positions.
```
Comment (timhodson) [inline topics/hierarchy-descendants.md]: Not just when in Beta. This is just sensible practice anyway as there is always the chance of adding new things.

File: topics/timeseries.md [hunk 7]
```diff
@@ -8,12 +8,24 @@ Timeseries data differs from the [raw]({{site.baseurl}}/topics/raw.html) data in
...
-
-Each sliding window has its own data retention policy [which is documented here]({{site.baseurl}}/topics/limits.html).
+There are several different views representing different sliding window lengths and history:
+
+1. `f_event_timeseries_1hr` 
+    - where the event counts and sums are per hour
+1. `f_event_timeseries_24hr` 
+    - where the event counts and sums are per day. 
+    - up to 36 months historical data
+1. `f_event_timeseries_24hr_last_3_months`
```
Comment (timhodson) [inline topics/timeseries.md]: Added detail about the other views...

File: topics/timeseries.md [hunk 8]
```diff
@@ -169,6 +197,28 @@ these columns depends on the `event_class` in question and is documented in deta
 | `dimension_4` | User's Globally Unique ID, can be joined to [`public.f_rl_users.talis_guid`]({{ site.baseurl }}/topics/users.html). | `myoVK7wfosXXWlw` |
 
 <br/>
+<a name="player-profile-complete"></a>
+
+### Events with class `player.profile.complete`
+
+`digitisation.download` is also recorded by the Talis Elevate player when the content has originated from Copyright Clearance. This is in addition to `player.download`.
```
Comment (lordtatty) [inline topics/timeseries.md]: I think this paragraph has been duplicated from above and not needed?

File: topics/event-classes.md [hunk 9]
```diff
@@ -6,32 +6,70 @@ body_class: event-classes
...
+| `player.search` | A user used the search function |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-search)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.download` | The player resource was downloaded by a user. `digitisation.download` is also recorded by the Talis Elevate player when the content has originated from Copyright Clearance. This is in addition to `player.download`. | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-download)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+|`digitisation.print`| A digitisation played through talis elevate was accessed for printing. This event is only emitted when a digitisation is played in Talis Elevate. | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-print)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```
Comment (camillef) [inline topics/event-classes.md]: I guess we should probably add `player.print` and the other missing player events to the config... but one for a follow up

File: topics/timeseries.md [hunk 10]
```diff
@@ -8,12 +8,24 @@ Timeseries data differs from the [raw]({{site.baseurl}}/topics/raw.html) data in
...
-1. `f_event_timeseries_1hr` where the event counts and sums are per hour
-1. `f_event_timeseries_24hr` where the event counts and sums are per day
-
-Each sliding window has its own data retention policy [which is documented here]({{site.baseurl}}/topics/limits.html).
+There are several different views representing different sliding window lengths and history:
+
+1. `f_event_timeseries_1hr` 
+    - where the event counts and sums are per hour
+1. `f_event_timeseries_24hr` 
+    - where the event counts and sums are per day.
```
Comment (camillef) [inline topics/timeseries.md]:
```suggestion
    - where the event counts and sums are per day
```

File: topics/timeseries.md [hunk 11]
```diff
@@ -66,6 +84,10 @@ these columns depends on the `event_class` in question and is documented in deta
 
 ### Events with class `addToList`
 
+This event is emitted when a user has added a bookmark to a list. Only in classic
```
Comment (camillef) [inline topics/timeseries.md]: This does also get submitted when a resource is added to a list straight from the user's my bookmarks page

File: topics/timeseries.md [hunk 12]
```diff
@@ -66,6 +84,10 @@ these columns depends on the `event_class` in question and is documented in deta
...
+
+<span class="deprecated">DEPRECATED</span> you will find `list.edit` to contain more useful information about what has been added to a list and when.
```
Comment (camillef) [inline topics/timeseries.md]: I'm not totally sure it's fair to deprecate this until we cover off the route to add bookmarks via the My Bookmarks page with a new event...

Comment (timhodson) [reply topics/timeseries.md]: Ok. left this as undeprecated.

**Pull request merged**